2025-06-22 17:11:08,646 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:11:08,841 - DEBUG - FILE - qb_odoo_sync - data_loader - load_field_mapping - 69 - Attempting to load field mapping from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:11:08,846 - INFO - FILE - qb_odoo_sync - data_loader - load_field_mapping - 85 - Successfully loaded field mapping from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:11:08,858 - INFO - FILE - qb_odoo_sync - odoo_service - _load_mappings - 37 - Field mapping loaded successfully in odoo_service: ['mapping_version', 'source_system', 'target_system', 'entities']
2025-06-22 17:11:08,859 - DEBUG - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 33 - Attempting to load account crosswalk from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:11:08,861 - INFO - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 49 - Successfully loaded 168 account mappings from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:11:08,861 - INFO - FILE - qb_odoo_sync - odoo_service - load_account_crosswalk - 753 - Ensured account crosswalk data is loaded via data_loader.
2025-06-22 17:11:08,866 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:11:08,910 - INFO - FILE - qb_odoo_sync - run - main - 34 - QuickBooks sync server starting on http://0.0.0.0:5000/quickbooks
2025-06-22 17:11:08,911 - INFO - FILE - qb_odoo_sync - run - main - 35 - Using Odoo URL: https://nterra22-sounddecision-odoo-develop-20178686.dev.odoo.com/
2025-06-22 17:11:08,911 - INFO - FILE - qb_odoo_sync - run - main - 36 - Flask DEBUG mode: True
2025-06-22 17:11:10,711 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:11:10,863 - DEBUG - FILE - qb_odoo_sync - data_loader - load_field_mapping - 69 - Attempting to load field mapping from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:11:10,876 - INFO - FILE - qb_odoo_sync - data_loader - load_field_mapping - 85 - Successfully loaded field mapping from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:11:10,884 - INFO - FILE - qb_odoo_sync - odoo_service - _load_mappings - 37 - Field mapping loaded successfully in odoo_service: ['mapping_version', 'source_system', 'target_system', 'entities']
2025-06-22 17:11:10,895 - DEBUG - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 33 - Attempting to load account crosswalk from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:11:10,899 - INFO - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 49 - Successfully loaded 168 account mappings from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:11:10,900 - INFO - FILE - qb_odoo_sync - odoo_service - load_account_crosswalk - 753 - Ensured account crosswalk data is loaded via data_loader.
2025-06-22 17:11:10,908 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:11:10,999 - INFO - FILE - qb_odoo_sync - run - main - 34 - QuickBooks sync server starting on http://0.0.0.0:5000/quickbooks
2025-06-22 17:11:11,012 - INFO - FILE - qb_odoo_sync - run - main - 35 - Using Odoo URL: https://nterra22-sounddecision-odoo-develop-20178686.dev.odoo.com/
2025-06-22 17:11:11,014 - INFO - FILE - qb_odoo_sync - run - main - 36 - Flask DEBUG mode: True
2025-06-22 17:11:25,701 - DEBUG - FILE - spyne.protocol.xml - xml - create_out_string - 632 - [1;31mResponse[0m b'<soap11env:Envelope xmlns:soap11env="http://schemas.xmlsoap.org/soap/envelope/">\n  <soap11env:Body>\n    <soap11env:Fault>\n      <faultcode>soap11env:Client.RequestNotAllowed</faultcode>\n      <faultstring>You must issue a POST request with the Content-Type header properly set.</faultstring>\n      <faultactor></faultactor>\n    </soap11env:Fault>\n  </soap11env:Body>\n</soap11env:Envelope>\n'
2025-06-22 17:11:25,701 - DEBUG - FILE - spyne.protocol.xml - xml - create_out_string - 632 - [1;31mResponse[0m b'<soap11env:Envelope xmlns:soap11env="http://schemas.xmlsoap.org/soap/envelope/">\n  <soap11env:Body>\n    <soap11env:Fault>\n      <faultcode>soap11env:Client.RequestNotAllowed</faultcode>\n      <faultstring>You must issue a POST request with the Content-Type header properly set.</faultstring>\n      <faultactor></faultactor>\n    </soap11env:Fault>\n  </soap11env:Body>\n</soap11env:Envelope>\n'
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - __validate_lxml - 419 - Validated ? True
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - __validate_lxml - 419 - Validated ? True
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 403 - [1;32mMethod request string:[0m {http://developer.intuit.com/}serverVersion
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 403 - [1;32mMethod request string:[0m {http://developer.intuit.com/}serverVersion
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 404 - b'<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">\n  <soap:Body>\n    <serverVersion xmlns="http://developer.intuit.com/"/>\n  </soap:Body>\n</soap:Envelope>\n'
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 404 - b'<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">\n  <soap:Body>\n    <serverVersion xmlns="http://developer.intuit.com/"/>\n  </soap:Body>\n</soap:Envelope>\n'
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - create_out_string - 632 - [1;31mResponse[0m b'<soap11env:Envelope xmlns:soap11env="http://schemas.xmlsoap.org/soap/envelope/" xmlns:tns="http://developer.intuit.com/">\n  <soap11env:Body>\n    <tns:serverVersionResponse>\n      <tns:serverVersionResult>1.0.0</tns:serverVersionResult>\n    </tns:serverVersionResponse>\n  </soap11env:Body>\n</soap11env:Envelope>\n'
2025-06-22 17:12:04,064 - DEBUG - FILE - spyne.protocol.xml - xml - create_out_string - 632 - [1;31mResponse[0m b'<soap11env:Envelope xmlns:soap11env="http://schemas.xmlsoap.org/soap/envelope/" xmlns:tns="http://developer.intuit.com/">\n  <soap11env:Body>\n    <tns:serverVersionResponse>\n      <tns:serverVersionResult>1.0.0</tns:serverVersionResult>\n    </tns:serverVersionResponse>\n  </soap11env:Body>\n</soap11env:Envelope>\n'
2025-06-22 17:12:06,171 - DEBUG - FILE - spyne.protocol.xml - xml - __validate_lxml - 419 - Validated ? True
2025-06-22 17:12:06,171 - DEBUG - FILE - spyne.protocol.xml - xml - __validate_lxml - 419 - Validated ? True
2025-06-22 17:12:06,178 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 403 - [1;32mMethod request string:[0m {http://developer.intuit.com/}clientVersion
2025-06-22 17:12:06,178 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 403 - [1;32mMethod request string:[0m {http://developer.intuit.com/}clientVersion
2025-06-22 17:12:06,206 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 404 - b'<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">\n  <soap:Body>\n    <clientVersion xmlns="http://developer.intuit.com/">\n      <strVersion>33.0.10015.91</strVersion>\n    </clientVersion>\n  </soap:Body>\n</soap:Envelope>\n'
2025-06-22 17:12:06,206 - DEBUG - FILE - spyne.protocol.xml - xml - validate_body - 404 - b'<soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">\n  <soap:Body>\n    <clientVersion xmlns="http://developer.intuit.com/">\n      <strVersion>33.0.10015.91</strVersion>\n    </clientVersion>\n  </soap:Body>\n</soap:Envelope>\n'
2025-06-22 17:12:06,212 - DEBUG - FILE - spyne.protocol.xml - xml - create_out_string - 632 - [1;31mResponse[0m b'<soap11env:Envelope xmlns:soap11env="http://schemas.xmlsoap.org/soap/envelope/" xmlns:tns="http://developer.intuit.com/">\n  <soap11env:Body>\n    <tns:clientVersionResponse>\n      <tns:clientVersionResult></tns:clientVersionResult>\n    </tns:clientVersionResponse>\n  </soap11env:Body>\n</soap11env:Envelope>\n'
2025-06-22 17:12:06,212 - DEBUG - FILE - spyne.protocol.xml - xml - create_out_string - 632 - [1;31mResponse[0m b'<soap11env:Envelope xmlns:soap11env="http://schemas.xmlsoap.org/soap/envelope/" xmlns:tns="http://developer.intuit.com/">\n  <soap11env:Body>\n    <tns:clientVersionResponse>\n      <tns:clientVersionResult></tns:clientVersionResult>\n    </tns:clientVersionResponse>\n  </soap11env:Body>\n</soap11env:Envelope>\n'
2025-06-22 17:13:23,324 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:13:23,424 - DEBUG - FILE - qb_odoo_sync - data_loader - load_field_mapping - 69 - Attempting to load field mapping from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:13:23,429 - INFO - FILE - qb_odoo_sync - data_loader - load_field_mapping - 85 - Successfully loaded field mapping from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:13:23,448 - INFO - FILE - qb_odoo_sync - odoo_service - _load_mappings - 37 - Field mapping loaded successfully in odoo_service: ['mapping_version', 'source_system', 'target_system', 'entities']
2025-06-22 17:13:23,449 - DEBUG - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 33 - Attempting to load account crosswalk from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:13:23,451 - INFO - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 49 - Successfully loaded 168 account mappings from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:13:23,452 - INFO - FILE - qb_odoo_sync - odoo_service - load_account_crosswalk - 753 - Ensured account crosswalk data is loaded via data_loader.
2025-06-22 17:13:23,468 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:13:23,483 - INFO - FILE - qb_odoo_sync - run - main - 34 - QuickBooks sync server starting on http://0.0.0.0:5000/quickbooks
2025-06-22 17:13:23,483 - INFO - FILE - qb_odoo_sync - run - main - 35 - Using Odoo URL: https://nterra22-sounddecision-odoo-develop-20178686.dev.odoo.com/
2025-06-22 17:13:23,483 - INFO - FILE - qb_odoo_sync - run - main - 36 - Flask DEBUG mode: True
2025-06-22 17:13:24,635 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:13:24,684 - DEBUG - FILE - qb_odoo_sync - data_loader - load_field_mapping - 69 - Attempting to load field mapping from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:13:24,684 - INFO - FILE - qb_odoo_sync - data_loader - load_field_mapping - 85 - Successfully loaded field mapping from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\field_mapping.json
2025-06-22 17:13:24,684 - INFO - FILE - qb_odoo_sync - odoo_service - _load_mappings - 37 - Field mapping loaded successfully in odoo_service: ['mapping_version', 'source_system', 'target_system', 'entities']
2025-06-22 17:13:24,684 - DEBUG - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 33 - Attempting to load account crosswalk from: C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:13:24,709 - INFO - FILE - qb_odoo_sync - data_loader - load_account_crosswalk - 49 - Successfully loaded 168 account mappings from C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\data\account_crosswalk.json
2025-06-22 17:13:24,709 - INFO - FILE - qb_odoo_sync - odoo_service - load_account_crosswalk - 753 - Ensured account crosswalk data is loaded via data_loader.
2025-06-22 17:13:24,718 - INFO - FILE - qb_odoo_sync - logging_config - setup_logging - 52 - File logging initialized. Debug messages will be written to C:\SD Code Files\QB Odoo Sync\QB_to_Odoo_Sync\qb_odoo_sync_project\logs\qbwc_debug.log
2025-06-22 17:13:24,755 - INFO - FILE - qb_odoo_sync - run - main - 34 - QuickBooks sync server starting on http://0.0.0.0:5000/quickbooks
2025-06-22 17:13:24,755 - INFO - FILE - qb_odoo_sync - run - main - 35 - Using Odoo URL: https://nterra22-sounddecision-odoo-develop-20178686.dev.odoo.com/
2025-06-22 17:13:24,755 - INFO - FILE - qb_odoo_sync - run - main - 36 - Flask DEBUG mode: True
